### ISSUE TITLE
[harbor-exporter] error upgrading to Harbor V2.3.0

### ISSUE THREAD

**Issue by dioguerra** [goharbor/harbor-helm#997]
I am seeing an error on the harbor-exporting after upgrading from V2.2.0 to V2.3.0.

```
$~/ws/helm/releases/cci-container-infra$ kn staging logs -f harbor-staging-harbor-exporter-7bf898854b-nwqn8
Appending internal tls trust CA to ca-bundle ...
find: '/etc/harbor/ssl': No such file or directory
Internal tls trust CA appending is Done.
2021-07-06T08:26:26Z [INFO] [/common/dao/base.go:65]: Registering database: type-PostgreSQL harbordb.foo.bar port-6601 database-registry sslmode-"require"
2021-07-06T08:26:26Z [INFO] [/common/dao/base.go:70]: Register database completed
panic: bad redis url: not support scheme 

goroutine 1 [running]:
github.com/goharbor/harbor/src/pkg/exporter.InitBackendWorker(0xc0001efec0)
	/harbor/src/pkg/exporter/js_worker.go:42 +0x265
main.main()
	/harbor/src/cmd/exporter/main.go:50 +0x7c5

```

It seems that the ssl mode is replaced in the connection configuration with the **"**
Maybe this is more suited to the goharbor/harbor repository?

**Comment by memogarcia** [comment]
@dioguerra Did you managed to fix this issue?

**Comment by dioguerra** [comment]
This was fixed in the 2.3.2 version if not mistaken

**Comment by memogarcia** [comment]
Yes, I used a new version of the chart and it worked (`1.8.1`)
